include/openssl/e_os2.h

```diff
@@ -350,6 +350,16 @@ typedef unsigned __int64 uint64_t;
 #  endif
 # endif
 
+# ifdef __STDC_VERSION__
+#  if (__STDC_VERSION__ >= 201112L)
+#   include <stdnoreturn.h>
+#  endif
```

Comment:
Let's not exagerate ;-) 
I'd think the first condition should be like this :

    #if defined(__STDC_VERSION__) && __STDC_VERSION__ >= 201112L

...unless I'm missing something.